src/navmazing/_step.py

```diff
@@ -124,6 +135,19 @@ def go(self, _tries: int = 0, *args: object, **kwargs: object) -> object:
             self.parent = self.prerequisite(*args, **kwargs)
             self.logger.info(f"NAVIGATE: Heading to destination {self._name}")
             self.do_nav(_tries, *args, **kwargs)
-        self.resetter(*args, **kwargs)
+        if kwargs.get("use_resetter", True):
```

Comment:
Kwarg defaults and kwonly args are supported, let's put them into the signature with type annotations